frontend/src/components/HandbookOverlay/HandbookSystemRole.test.tsx

```diff
@@ -121,4 +124,41 @@ describe('HandbookSystemRole component tests', () => {
 		const lockedBox = container.getElementsByClassName('role-locked');
 		expect(lockedBox).toHaveLength(0);
```

Comment:
@dcottle-scottlogic This is also a no-no, testing to see if classnames have been used. Again, not something you added! I wonder if we might pair on this to correct things in this test file. I can explain as we go.

Reply:
@chriswilty Yeah sure can do 👍